aptos-move/block-executor/src/executor.rs

```diff
@@ -482,25 +482,26 @@ where
                 }
 
                 // Either all txn committed, or a committed txn caused an early halt.
-                if scheduler.halt() {
```

Comment:
Unrelated change?

Reply:
I've decided to revert to previous stage - before it was broken (otherwise I don't get any counters at all), and @ziaptos / @danielxiangzl  can fix it properly

Reply:
oh that got landed- let me rebase